spec/features/admin/invoices/show_spec.rb

```diff
@@ -33,5 +33,24 @@
         expect(page).to have_content("Status: #{alaina_invoice1.status}")
         expect(page).to have_content("Created at: #{alaina_invoice1.created_at.strftime("%A, %B %d, %Y")}")
         expect(page).to have_content("Customer: #{alaina.name}")
+        expect(page).to_not have_content("Invoice ##{alaina_invoice2.id}")
+    end
+
+    describe 'invoice items' do
+        it 'shows all invoice items' do
+            visit admin_invoice_path(alaina_invoice1)
+            expect(page).to have_content(alainainvoice1_itemgold_earrings.item.name)
+            expect(page).to have_content(alainainvoice1_itemsilver_necklace.item.name)
+            expect(page).to_not have_content(alainainvoice2_itemstudded_bracelet.item.name)
+        end
+
+        it 'shows all item info' do
+            visit admin_invoice_path(alaina_invoice1)
+            expect(page).to have_content(alainainvoice1_itemgold_earrings.item.name)
```

Comment:
This is a redundant test, nbd at all just maybe something to consider refactoring out in the future